src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/resolver/VersionData.kt

```diff
@@ -187,7 +188,7 @@ fun parseVersionData(json: JsonObject): VersionData {
                 .toEpochMilli()
         } ?: 0,
         json.get("minimumLauncherVersion")?.asInt ?: 0,
-        json.get("mainClass").asString,
+        json.get("mainClass").let { if (it is JsonNull) null else it.asString },
```

Comment:
`?.let`

Reply:
Sorry, I intended to do exactly that, but mistyped apparently. Fixed!